saleor/graphql/product/mutations/products.py

```diff
@@ -743,16 +747,55 @@ def _validate_input(
         else:
             return cls._check_input_for_product(cleaned_input, attribute_qs)
 
+    @classmethod
+    def _validate_page_attributes_input(
```

Comment:
This function is the same as `_check_input_for_product`. Maybe we could just move the logic from this function to `validate_attributes_input_for_product_and_page`? It looks like some validation is done here and some in the utils functions. It would be better to have validation in one place. You can take a look if we can fix it somehow.